libs/vscode/tasks/src/lib/cli-task-commands.ts

```diff
@@ -49,30 +47,30 @@ export function registerCliTaskCommands(
     );
   });
 
-  commands.registerCommand('nx.run', () =>
-    selectCliCommandAndPromptForFlags('run')
-  );
-  commands.registerCommand(`nx.run.fileexplorer`, (uri: Uri) =>
-    selectCliCommandAndPromptForFlags('run', getCliProjectFromUri(uri))
-  );
+  ['ng', 'nx'].forEach(cli => {
+    commands.registerCommand(
+      `${cli}.run`,
+      (project?: string, target?: string, configuration?: string) => {
+        selectCliCommandAndPromptForFlags('run', project, target, configuration)
+      }
+    );
+    commands.registerCommand(`${cli}.run.fileexplorer`, (uri: Uri) =>
+      selectCliCommandAndPromptForFlags('run', getCliProjectFromUri(uri))
+    );
 
-  commands.registerCommand(`ng.generate`, () =>
-    selectSchematicAndPromptForFlags()
-  );
+    commands.registerCommand(`${cli}.generate`, () =>
+      selectSchematicAndPromptForFlags()
+    );
 
-  commands.registerCommand(`ng.generate.ui`, () =>
-    selectCliCommandAndShowUi('generate', context.extensionPath)
-  );
-  commands.registerCommand(`nx.generate`, () =>
-    selectSchematicAndPromptForFlags()
-  );
+    commands.registerCommand(`${cli}.generate.ui`, () =>
+      selectCliCommandAndShowUi('generate', context.extensionPath)
+    );
+
+    commands.registerCommand(`${cli}.generate.ui.fileexplorer`, (uri: Uri) =>
```

Comment:
The context menu generate and run commands were also just implemented in the `Nx` category. All of our other commands are implemented both as `nx` and `ng`.
* Right click `Nx generate (ui)` was working on Angular CLI projects, but it was just an "nx" VS Code command. 
* Adding a separate command for `ng.generate.ui.fileexplorer` gives us finer control over that menu item, so I made it "ng generate (ui)" for Angular CLI projects.
* I also added `ng run` to the context menu for Angular CLI projects that uses the Command Palette and prompts